app.py

```diff
@@ -32,6 +32,23 @@
 app.secret_key = os.environ['SECRET_KEY']
 app.wtf_csrf_secret_key = os.environ['WTF_CSRF_SECRET_KEY']
 
+SNAPCRAFT_IO_API = os.environ.get(
```

Comment:
I do have a preference to using `os.getenv()`, which is specifically created for this use.